docs_user/modules/cinder-cfg.py

```diff
@@ -0,0 +1,765 @@
+#!/usr/bin/env python
```

Comment:
How will this script be used in downstream context? I remember at some point we discussed whether we should be using just OS Diff for everything and phase this script out but i can't find what the result of the discussion was.

Reply:
I don't know if it will be used or not.
I know we will not be providing an RPM for it or anything like that.

I wasn't part of the OS Diff discussion, and I haven't tested that tool against the various cinder configuration scenarios I tried this script with back then.

Reply:
If there's no intention to ship the tool, we should remove it and reformulate the docs to not depend on it?

Reply:
I'm not sure if we can just point to a Github script from downstream docs...

Reply:
I don't know if we can or cannot, but what we have in this upstream repo is broken, so I was just looking for a quick fix right now.

Reply:
Ack i think we can merge this for now (the PR also contains other tweaks).

We also discussed today in UA/TC/PM sync for Upgrades -- we could keep this for upstream only in theory, but we need to get rid of mentioning anything in the downstream docs render that we do not intend to ship. The downstream/upstream rendering can be differentiated via if/else branching as seen here for example:

https://github.com/openstack-k8s-operators/data-plane-adoption/pull/286

But my 2 cents would be to keep the upstream/downstream instructions as close as possible, which probably means not relying on this script in documentation. IMO there is limited utility to anything we don't intend to ship and support...

I'll create a bug for tracking.